src/player.ts

```diff
@@ -448,4 +448,14 @@ export class Player extends NativeInstance<PlayerConfig> {
   getMaxTimeShift = async (): Promise<number> => {
     return PlayerModule.getMaxTimeShift(this.nativeId);
   };
+
+  /**
+   * Sets the upper bitrate boundary for video qualities. All qualities with a bitrate
+   * that is higher than this threshold will not be eligible for automatic quality selection.
+   *
+   * Can be set to `undefined` for no limitation.
+   */
+  setMaxSelectableBitrate = (bitrate: number | undefined) => {
+    PlayerModule.setMaxSelectableBitrate(this.nativeId, bitrate || -1);
```

Comment:
Can we pass null to the PlayerModule? Then we wouldn't need to do any special handling here and in the native parts we would just work with nulls (which would also be a lot cleaner IMO).

I see that Android's native part is already nullable, but the iOS part isn't, is there a reason for that?

Reply:
@zigavehovec actually that's a mistake from my side.
I planned having this as nullable but when making the iOS side nullable, the React Native runtime throws an error that it can not be nullable to keep it compatible with Android. Don't ask me why, maybe a Java restriction with JS interop?

Removed the nullability from Android here: b88b5de

Reply:
I see, thanks for the clarification!